docs/AboutUs.md

```diff
@@ -13,53 +13,53 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 <img src="images/aweijun.png" width="200px">
 
-[github](https://github.com/aweijun)
+[<img src="images/github-icon.png" width="50px">](https://github.com/aweijun) &nbsp; &nbsp; &nbsp;
+[<img src="images/portfolio-icon.png" width="50px">](team/aweijun.md)
 
-[portfolio](team/aweijun.md)
-
-* Role: Developer
-* Responsibilities: Integration
+|   Role    | Responsibilities |
+|:---------:|:-----------------|
+| Developer | Integration      |
 
 ### Jonathan Yeo
 
 <img src="images/jonathanhoshi.png" width="200px">
 
-| [GitHub](http://github.com/jonathanhoshi)
-| [Portfolio](team/jonathanhoshi.md) 
-|
+[<img src="images/github-icon.png" width="50px">](http://github.com/jonathanhoshi) &nbsp; &nbsp; &nbsp;
+[<img src="images/portfolio-icon.png" width="50px">](team/jonathanhoshi.md)
 
-* Role: Developer
-* Responsibilities: 
-  * Team Lead
-  * UI
+|   Role    | Responsibilities   |
+|:---------:|:-------------------|
+| Developer | Team Lead <br/> UI |
```

Comment:
Should we reorder Jon to first if he taking team lead?

Reply:
Yeap, I agree also, makes sense for the team lead to be at the top!

Reply:
Noted.